src/components/v5/frame/ColonyHome/partials/ReputationChart/partials/LegendItem.tsx

```diff
@@ -0,0 +1,38 @@
+import clsx from 'clsx';
+import React, { type FC } from 'react';
+
+import Numeral from '~shared/Numeral/Numeral.tsx';
+
+import { summaryLegendColor } from '../consts.ts';
+import { type ReputationChartDataItem } from '../types.ts';
+
+const displayName = 'v5.frame.ColonyHome.ReputationChart.partials.LegendItem';
+interface LegendItemProps {
+  chartItem:
+    | ReputationChartDataItem
+    | { color: string; label: string; value: undefined };
+}
+
+const LegendItem: FC<LegendItemProps> = ({
+  chartItem: { color, label, value },
+}) => {
+  return (
+    <div className="flex flex-row items-center gap-1">
+      <div
+        className={clsx(
+          'h-[10px] w-[10px] rounded-full',
+          summaryLegendColor[color] || summaryLegendColor.default,
+        )}
+      />
+      <span className="text-xs font-normal text-gray-900">{label}</span>
```

Comment:
Here we should use `text-gray-500`

![Screenshot 2024-08-30 at 18 20 29](https://github.com/user-attachments/assets/64d0d9de-1976-403c-87cf-54dfaebb213d)